src/Controller/CivicrmController.php

```diff
@@ -29,17 +29,9 @@ public static function create(ContainerInterface $container) {
     );
   }
 
-  public function main($args, $one, $two, $three, $four, $five) {
```

Comment:
@totten After i reverted this and applied a small patch to the crmNavigationMenu smarty function i got the civicrm menu reliably loading